sdk/test/logs/batch_log_processor_test.cc

```diff
@@ -9,6 +9,8 @@
 
 #  include <gtest/gtest.h>
 #  include <chrono>
+#  include <list>
```

Comment:
Are these 2 new headers necessary as the tests are not changed?

Reply:
Thanks and this is removed now.